reVX/wind_setbacks/wind_setbacks.py

```diff
@@ -0,0 +1,1386 @@
+            rotor_diameter, regs_fpath=None, multiplier=None,
+            chunks=(128, 128), max_workers=None, description=None,
+            replace=False):
+        """
+        Compute road setbacks and write them as a new layer to the
+        exclusions .h5 file. If a wind regulations file is given compute
+        existing setbacks, otherwise compute general setbacks using the given
+        multiplier and the turbine tip-height. File must be locally on disc to
+        allow for writing of new layer.
+
+        Parameters
+        ----------
+        excl_h5 : str
+            Path to .h5 file containing or to contain exclusion layers
+        roads_dir : str
+            Path to directory containing here streets *.gdb files. Used
+            to identify roads to build setbacks from. Files should be
+            by state
+        layer_name : str
+            Name of new layer to write to exclusions .h5 file containing
+            computed setbacks
+        hub_height : float | int
+            Turbine hub height (m), used along with rotor diameter to compute
+            blade tip height which is used to determine setback distance
+        rotor_diameter : float | int
+            Turbine rotor diameter (m), used along with hub height to compute
+            blade tip height which is used to determine setback distance
+        regs_fpath : str | None, optional
+            Path to wind regulations .csv file, if None create global
+            setbacks, by default None
+        multiplier : int | float | str | None, optional
+            setback multiplier to use if wind regulations are not supplied,
+            if str, must one of {'high': 3, 'moderate': 1.1},
+            by default None
+        chunks : tuple, optional
+            Chunk size to use for setback layers, if None use default chunk
+            size in excl_h5, by default (128, 128)
+        max_workers : int, optional
+            Number of workers to use for setback computation, if 1 run in
+            serial, if > 1 run in parallel with that many workers, if None
+            run in parallel on all available cores, by default None
+        description : str, optional
+            Description of exclusion layer (set as an attribute),
+            by default None
+        replace : bool, optional
+            Flag to replace existing layer data with arr if layer already
+            exists in the exlcusion .h5 file, by default False
+        """
+        setbacks = cls(excl_h5, hub_height, rotor_diameter,
+                       regs_fpath=regs_fpath, multiplier=multiplier,
+                       hsds=False, chunks=chunks)
+        setbacks.compute_setbacks(roads_dir, layer=layer_name,
+                                  max_workers=max_workers,
+                                  description=description,
+                                  replace=replace)
+
+
+class TransmissionWindSetbacks(BaseWindSetbacks):
+    """
+    Transmission Wind setbacks
```

Comment:
without having to do a code diff can you explain how/why some of these transmission setbacks and methods are different than the structure setbacks?